cloud/ovirt/ovirt_disks.py

```diff
@@ -199,6 +204,7 @@ def update_check(self, entity):
             equal(self._module.params.get('description'), entity.description) and
             equal(convert_to_bytes(self._module.params.get('size')), entity.provisioned_size) and
             equal(self._module.params.get('format'), str(entity.format)) and
+            equal(self._module.params.get('sparse'), entity.sparse) and
```

Comment:
Currently it's not possible to change allocation policy of the disk in API. so this shouldn't be here, as we can't update it.